mlflow/projects/__init__.py

```diff
@@ -59,20 +59,30 @@ def _resolve_experiment_id(experiment_name=None, experiment_id=None):
 
     Verifies either one or other is specified - cannot be both selected.
 
+    If `experiment_name` is provided and does not exist, an experiment
```

Comment:
double-backticks please (it's hard to convey this in github comments because it tries to format them... aha,
```
``experiment_name``
```
is what we need